src/components/ContentPage.js

```diff
@@ -51,12 +63,14 @@ export class ContentPage extends Component {
         <Spinner loading={this.props.loading} className="spinner" size={100} />
 
         {this.props.publications.length > 0 && <PubsTable publications={this.props.publications} /> }
+        
 
-
-        <h3 className="word-cloud-title pt-4 mt-4"> What are these publications all about? </h3>
+        {this.props.publications.length > 0 && <h3 className="word-cloud-title pt-4 mt-4"> What are these publications all about? </h3>}
```

Comment:
I know we had previously discussed having multiple spinners, but I think it might be cleaner to use one `this.props.loading` as a gatekeeper to all of this content, so there would just be one `Spinner` and then everything from line 65-74(ish) would be inside a single div that is conditionally rendered on `!this.props.loading`